src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -929,6 +929,13 @@ export default {
 }
 </script>
 
+<style lang="scss">
+// Tribute container inherits box-sizing: border-box from NcContent, so height should be overrided
+.tribute-container .autocomplete-result {
+	height: 64px !important;
+}
+</style>
```

Comment:
If the problem is different `box-sizing`, cannot we set only correct `box-sizing` value?

Reply:
Thought about it. Will try that later